website/views/shared/header/header.jade

```diff
@@ -12,23 +12,25 @@
       .meter.health(tooltip='{{Math.round(user.stats.hp * 100) / 100}}')
         .bar(ng-style='{"width": Shared.percent(user.stats.hp, Shared.maxHealth)+"%"}')
         span.meter-text.value
-          | {{Math.ceil(user.stats.hp)}} / {{::Shared.maxHealth}}
+          | {{statCalc.hpDisplay(user.stats.hp)}}
       .meter-label(tooltip=env.t('experience'))
         span.glyphicon.glyphicon-star
       .meter.experience(tooltip='{{Math.round(user.stats.exp * 100) / 100}}')
         .bar(ng-style='{"width": Shared.percent(user.stats.exp,Shared.tnl(user.stats.lvl))+"%"}')
         span.meter-text.value
+          span(ng-show='statCalc.canLevelUp(user)' class="hidden-xs") Complete a Task to Level Up!
```

Comment:
I tried putting this in a `bootstrap label`, and tried a few variations: `label-success`, `label-info`, `label-default` etc. but they didn't look good so I stuck with just plain text.